docs/upgrade-guides/3-13-to-3-14.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Upgrading from 3.13 to 3.14
+sidebar_position: 7
+---
+
+This migration guide describes the upgrade from versions 3.13 to 3.14. Version 3.14 contains the following breaking changes:
+
+## Plugins manager:
+
+In Saleor 3.14, we made the decision to denormalize sales applied to checkout. We extended `CheckoutLineInfo` by adding `discounts: List["CheckoutLineDiscount"]`. This field contains the sales that are applied to the lines. As a consequence of this change, the `discounts` field has been removed from the listed plugin manager methods. Instead of the `discounts` argument, an applied discount is now assigned to each line in the `CheckoutLineInfo` inside the `CheckoutInfo` object.
+
```

Comment:
IMO we should add a sentence before listing the related methods. Something like:
`This change applies to the following methods:`